examples/convert_eeg_to_bids.py

```diff
@@ -85,7 +85,7 @@
 # We will do exactly that in the next step.
 
 ###############################################################################
-# Formatting to BIDS
+# Converting to BIDS
```

Comment:
Argh sorry! To be consistent with the other headers, it should be phrased like this:
```suggestion
# Convert to BIDS
```

Reply:
good point, thanks!